src/lib/github/getFilesOnInstall.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Gets files that have changed in the last  commits
```

Comment:
Maybe comment should change. I think it should be something like `Get all repo files.`